src/pages/Register.jsx

```diff
@@ -53,13 +61,13 @@ function Register() {
         />
       </div>
       <div className="form-div">
-        <label htmlFor="password">Confirmation du mot de passe :</label>
+        <label htmlFor="confirmPassword">Confirmation du mot de passe :</label>
         <input
           className="border border-black"
-          ref={passwordInput}
+          ref={confirmPasswordInput}
           type="password"
-          name="password"
-          id="password"
+          name="confirmPassword"
+          id="confirmPassword"
```

Comment:
you added an input confirmPassword, but remove the input for the password: we need both

Reply:
this is not resolved. did you push your work?

Reply:
Fixed in the last commit